### PR TITLE
    OPS LDAP support, refactoring

### PR DESCRIPTION
**Pull request by naumvd95**
- refactor env-vars arch to provide configuration using only docker env-vars
    - remove apply_env changes due issues with bad overriding in case of multiple value-entries in config file
    - logging increased

### CODE REVIEW

File: kqueen_ui/config/utils.py [hunk 1]
```diff
@@ -17,34 +16,12 @@ def select_file(config_file=None):
...
 
-    if not config_file or config_file == 'None':
-        config_file = os.environ.get('KQUEEN_UI_CONFIG_FILE')
-        logger.debug('Config file from env variable: {}'.format(config_file))
-
-    if not config_file or config_file == 'None':
-        config_file = CONFIG_FILE_DEFAULT
-        logger.debug('Config file, using default: {}'.format(config_file))
+    CONFIG_FILE_DEFAULT = 'config/dev.py'
+    config_file = os.environ.get('KQUEEN_CONFIG_FILE', CONFIG_FILE_DEFAULT)
```
[the remark this answers is not in the document]
Comment (naumvd95) [reply kqueen_ui/config/utils.py]: done

File: kqueen_ui/api.py [hunk 2]
```diff
@@ -97,11 +97,11 @@ def login(self):
...
-            error = {
-                'status': r.status,
-                'description': msg
-            }
-            logger.warning('KQueen Client:: Could not get access token: {}'.format(r.error))
+                error = {
+                    'status': r.status,
+                    'description': msg
+                }
+                logger.exception('KQueen Client: Could not get access token: {}'.format(r.error))
```
[the remark this answers is not in the document]
Comment (naumvd95) [reply kqueen_ui/api.py]: done

File: kqueen_ui/config/test.py [hunk 3]
```diff
@@ -1,28 +1,48 @@
...
+    KQUEEN_SERVICE_USER_PASSWORD = os.environ.get('KQUEENUI_KQUEEN_SERVICE_USER_PASSWORD', 'default')
+    MAIL_SERVER = os.environ.get('KQUEENUI_MAIL_SERVER', 'mail')
+    MAIL_PORT = os.environ.get('KQUEENUI_MAIL_PORT', '10025')
+    ENABLE_PUBLIC_REGISTRATION = os.environ.get('KQUEENUI_ENABLE_PUBLIC_REGISTRATION', True)
+
+    # Auth configuration
+
+    # Enable email notifications to user
+    LDAP_AUTH_NOTIFY = os.environ.get('KQUEENUI_LDAP_AUTH_NOTIFY', False)
+    LOCAL_AUTH_NOTIFY = os.environ.get('KQUEENUI_LOCAL_AUTH_NOTIFY', False)
```
[the remark this answers is not in the document]
Comment (naumvd95) [reply kqueen_ui/config/test.py]: done

File: kqueen_ui/config/utils.py [hunk 4]
```diff
@@ -17,34 +16,12 @@ def select_file(config_file=None):
         str: filename to be used as a configuration file
     """
 
-    if not config_file or config_file == 'None':
-        config_file = os.environ.get('KQUEEN_UI_CONFIG_FILE')
```
Comment (katyafervent) [inline kqueen_ui/config/utils.py]: KQUEEN_UI_CONFIG_FILE -> KQUEENUI_CONFIG_FILE

File: kqueen_ui/config/utils.py [hunk 5]
```diff
@@ -17,34 +16,12 @@ def select_file(config_file=None):
...
-    if not config_file or config_file == 'None':
-        config_file = CONFIG_FILE_DEFAULT
-        logger.debug('Config file, using default: {}'.format(config_file))
+    CONFIG_FILE_DEFAULT = 'config/dev.py'
+    config_file = os.environ.get('KQUEEN_UI_CONFIG_FILE', CONFIG_FILE_DEFAULT)
 
     return config_file
 
 
-def apply_env_changes(config, prefix='KQUEENUI_'):
```
Comment (katyafervent) [inline kqueen_ui/config/utils.py]: this function works correctly, please remove as it was.

File: kqueen_ui/config/utils.py (hunk 4 above)
Comment (naumvd95) [reply kqueen_ui/config/utils.py]: take a look on previous code, it was still equal @katyafervent

**Pull request closed without merging**